RetroBar/Utilities/Settings.cs

```diff
@@ -376,6 +383,14 @@ public enum TaskMiddleClickOption
         CloseTask
     }
 
+    public enum ClockClickOption
+    {
+        DoNothing,
+        OpenAeroClockFlyout,
+        OpenActionCenter,
+        OpenModernCalendar,
```

Comment:
I'd like to re-order this (both here and in the language). Could you swap the position of OpenActionCenter and OpenModernCalendar? The reason is that the modern calendar is more closely related to the aero clock flyout (i.e., they're both a calendar, not a notification list), and I'd like the order of them in the settings dropdown to reflect this.

Reply:
I was going to do it that way at first, just to find out that there was an Action Center in Windows Vista up to 8.1 (which I sometimes hid), I tried to open it with that option and move it to the side so I only had to remove the last item, but I'll reorder it as you say.